packages/arb-validator/ethbridge/globalInbox.go

```diff
@@ -63,6 +65,44 @@ func (con *globalInbox) SendTransactionMessage(ctx context.Context, data []byte,
 	return con.waitForReceipt(ctx, tx, "SendTransactionMessage")
 }
 
+func (con *globalInbox) DeliverTransactionBatch(
+	ctx context.Context,
+	chain common.Address,
+	transactions []message.Transaction,
+	signatures [][65]byte,
+) error {
+	con.auth.Lock()
```

Comment:
Could move the lock acquisition down to just before the DeliverTransactionBatch call, so the preparation for the call is done without holding the lock.

Reply:
Done